app/lib/constants/settings.rb

```diff
@@ -9,5 +9,8 @@ module Settings
       Organization
       Extensions
     ].freeze
+
+    VALID_URL = %r{\A(http|https)://([/|.\w\s-])*.[a-z]{2,5}(:[0-9]{1,5})?(/.*)?\z}.freeze
```

Comment:
the ultimate URL regexp :D

Reply:
It just moved from `SiteConfig` because I need to reference it in several places now.